superset/key_value/api.py

```diff
@@ -122,9 +113,7 @@ def put(self, pk: int, key: str) -> Response:
 
     def get(self, pk: int, key: str) -> Response:
         try:
-            args = CommandParameters(
-                actor=g.user, resource_id=pk, key=key, query_params=request.args
-            )
+            args = CommandParameters(actor=g.user, resource_id=pk, key=key)
             value = self.get_get_command()(args).run()
             if not value:
```

Comment:
What happens if the cache temporarily unavailable? we should set cache if cache broke down.

Reply:
Great call! I handled this problem in https://github.com/apache/superset/pull/18624